scenario/consistency_checker.py

```diff
@@ -24,6 +24,15 @@
 logger = scenario_logger.getChild("consistency_checker")
 
 
+def get_all_relations(charm_spec: "_CharmSpec"):
+    nonpeer_relations_meta = chain(
+        charm_spec.meta.get("requires", {}).items(),
+        charm_spec.meta.get("provides", {}).items(),
+    )
+    peer_relations_meta = charm_spec.meta.get("peers", {}).items()
+    return list(chain(nonpeer_relations_meta, peer_relations_meta))
```

Comment:
I don't understand why nonpeer and peer are done in two separate operations, rather than just `chain(required, provides, peers)`. What about avoiding chain and simplifying with something like:

```python
relations = [
    list(charm_spec.meta.get("requires", {}).items()) +
    list(charm_spec.meta.get("provides", {}).items()) +
    list(charm_spec.meta.get("peers", {}).items())
]
return relations
```

Reply:
hasty copy-paste from another piece of code where I needed them separately